code/src/cljs/sixsq/nuvla/ui/routing/routes.cljs

```diff
@@ -0,0 +1,179 @@
+      :link-text "home"}]
+    ["dashboard"
+     {:name      :dashboard
+      :view      dashboard-view
+      :link-text "dashboard"}]
+    ["apps"
+     {:name      :apps
+      :view      app-views/AppsOverview
+      :link-text "Apps"}
+     [""]
+     ["/" :apps-slashed]
+     ["/*sub-path"
+      {:name :apps-details
+       :view app-views/AppDetails}]]
+    ["deployments"
+     {:name      :deployments
+      :view      deployments-view
+      :link-text "deployments"}]
+    ["deployment"
+     {:name      :deployment
+      :view      deployments-view
+      :link-text "deployments"}]
+    ["deployment/"
+     {:name      :deployment-slashed
+      :view      deployments-view
+      :link-text "deployments"}]
+    ["deployment/:uuid"
+     {:name :deployment-details
+      :view DeploymentDetails}]
+    ["deployment-sets"
+     {:name      :deployment-sets
+      :view      deployment-sets-view
+      :link-text "deployment-sets"}
+     [""]
+     ["/" :deployment-sets-slashed]]
+    ["deployment-sets/:uuid"
+     {:name      :deployment-sets-details
+      :view      deployment-sets-view
+      :link-text "deployment-sets"}]
+    ["edges"
+     {:name      :edges
+      :view      edges-view
+      :link-text "edges"}
+     [""]
+     ["/" :edges-slashed]]
+    ["edges/:uuid"
+     {:name      :edges-details
+      :view      edges-view
+      :link-text "edges-details"}]
+    ["edges/nuvlabox-cluster/:uuid"
+     {:name :edge-cluster-details
+      :view edges-view}]
+    ["credentials"
+     {:name      :credentials
+      :view      credentials-view
+      :link-text "credentials"}
+     [""]
+     ["/" :credentials-slash]]
+    ["notifications"
+     {:name      :notifications
+      :view      notifications-view
+      :link-text "notifications"}]
+    ["data"
+     {:name      :data
+      :view      data-view
+      :link-text "data"}]
+    ["data/*uuid"
+     {:name :data-details
+      :view data-set-views/DataSet}]
+    ["clouds"
+     {:name      :clouds
+      :view      clouds-view
+      :link-text "clouds"}
+     [""]
+     ["/" :clouds-slashed]
+     ["/:uuid"
+      {:name      :clouds-details
+       :view      clouds-view
+       :link-text "clouds"}]]
+    ["documentation"
+     {:name      :documentation
+      :view      documentation
+      :link-text "documentation"}
+     [""]
+     ["/*sub-path" :documentation-sub-page]]
+    ["api"
+     {:name      :api
+      :view      api-view
+      :link-text "api"}
+     [""]
+     ["/" :api-slashed]
+     ["/*sub-path"
+      {:name :api-sub-page}]]
+    ["profile"
+     {:name :profile
+      :view profile}]]
+   ["/*sub-path"
+    {:name :catch-all
+     :view UnknownResource}]])
+
+(def router
+  (rf/router
+    r-routes
+    {:data      {:coercion rss/coercion}
+     :router    r/linear-router
+     :conflicts (fn [conflicts]
+                  (when debug?
+                    (println (exception/format-exception :path-conflicts nil conflicts))))}))
```

Comment:
this println is always fired-up at first load of the brother because routes contains conflicts.